app/themes/baseTheme.js

```diff
@@ -225,6 +225,7 @@ const text = {
   mediumTitle: {
     ...titleText,
     fontWeight: 'medium',
+    fontSize: 3
```

Comment:
I searched through blip and it looks like all instances of `<MediumTitle>` are used within the `<DialogTitle>`, meaning that this is somewhat a dialog-specific configuration. So I think that this is okay, as opposed to spinning off a new `variant`.